sources/web/datalab/polymer/modules/utils/utils.ts

```diff
@@ -32,6 +32,7 @@ class Utils {
     directory: 'folder',
     editorUrlComponent:   '/editor/',
     file: 'file',
+    me: 'Me',
```

Comment:
The Drive UI uses lower case 'me'. We could choose to be consistent with that.

Reply:
Done.